shell/platform/embedder/embedder.cc

```diff
@@ -821,6 +835,27 @@ FlutterEngineResult FlutterEngineRun(size_t version,
             texture.user_data          // texture release context
         );
 
+        /// Scale to the bounds size to adapt the user to change the Texture
```

Comment:
I'm having a hard time parsing this sentence; I'm not sure what "to adapt the user to change [...]" means.